### PR TITLE
Run swift-format on all files

### PR DESCRIPTION
**Pull request by mplorentz**
This is a 10 minute experiment I did after seeing this over the weekend and chatting about it with Itunu. I ran Apple's `swift-format` tool which has been around for a while but is now shipped with Xcode 16. It mostly made whitespace changes, but there are a few conventions it changed.

The exact command I ran was `swift format -ri .` after making a few tweaks in `.swift-format`.

I think we should probably start using this at some point because it will catch some small issues that we spend time on in code reviews. I'm not sure if now is the right time though. Probably after launch. But I'm curious to get everyone's quick thoughts if you want to spend 5 or 10 minutes looking over this.

### CODE REVIEW

File: Nos/Controller/ContentWarningController.swift [hunk 1]
```diff
@@ -39,20 +39,21 @@ enum ContentWarningType {
...
     private var firstAuthorSafeName: String {
         authorNames.first ?? String(localized: .localizable.unknownAuthor)
     }
-    
+
     /// The string explaining the reason(s) for the reports.
     private var reason: String {
-        let reasons = uniqueReasons
+        let reasons =
+            uniqueReasons
```
Comment (mplorentz) [inline Nos/Controller/ContentWarningController.swift:56]: hm, I like the way we had it before better, but I could get used to this.

File: Nos/Controller/FetchRequestPublisher.swift [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
         self.internalFRCP.publisher.replaceError(with: []).eraseToAnyPublisher()
     }()
 }
 
 // swiftlint:disable:next type_name
 private final class FetchedResultsControllerPublisherInternal<FetchType>: NSObject, NSFetchedResultsControllerDelegate
-    where FetchType: NSFetchRequestResult {
-    
+where FetchType: NSFetchRequestResult {
+
```
Comment (mplorentz) [inline Nos/Controller/FetchRequestPublisher.swift:34]: I have never seen this convention before.

File: Nos/Controller/NoteWarningController.swift [hunk 3]
```diff
@@ -31,94 +31,98 @@ extension Publisher {
...
                 FetchedResultsControllerPublisher(fetchedResultsController: noteReportsWatcher)
                     .publisher
                     .asyncMap { (events: [Event]?) -> [Event] in
                         var eventsFromFollowedAuthors = [Event]()
-                        for event in events ?? [] where
-                        await self.currentUser.socialGraph.follows(event.author?.hexadecimalPublicKey) {
+                        for event in events ?? []
+                        where
+                            await self.currentUser.socialGraph.follows(event.author?.hexadecimalPublicKey)
+                        {
```
[the remark this answers is not in the document]
Comment (joshuatbrown) [reply Nos/Controller/NoteWarningController.swift:94]: This entire expression grates on me. 😀 It's too long -- it should be broken up. I wonder what the formatter would do if it were already in multiple lines.

**Review by joshuatbrown — COMMENTED** [review]
Agreed that we should wait until after launch. I think I like this, though, and if we can tweak it a bit I think it could be perfect for us.

File: Nos/AppDelegate.swift
Comment (pelumy) [inline Nos/AppDelegate.swift:1]: uuuuh, I see what it did here, alphabetical order. Nice ❤️

**Comment by mplorentz** [comment]
@bryanmontz let us know your thoughts when you get a chance, but I'm closing this for now.

**Pull request closed without merging**